web/src/lib/components/shared-components/notification/notification.ts

```diff
@@ -9,9 +9,14 @@ export class ImmichNotification {
 	id = new Date().getTime();
 	type!: NotificationType;
 	message!: string;
+	action!: NotificationAction;
```

Comment:
Maybe we can use the property, `dismissible?: boolean` to indicate that the notification card can be dimissed by the user? Then in the `notification-card.svelte` you can use the `{#if} {/if}` syntax to show the dimiss button/icon

Reply:
Are there any notifications that we would not want to be dismissible?

Reply:
I think most of the info should not be dismissible, any notification card that might pop up constantly should have this flag enable, like error notification which might have multiple popup in a row

Reply:
I would argue for notifications being dismissible by default instead. Especially on the main gallery view, I can imagine wanting to get rid of a card so you can click something that's underneath. Especially if there are multiple (of any type), waiting for it to disappear could get annoying.

Reply:
That works too, so we have the `dismissible` property is true by default, is that how you are thinking?

Reply:
Fair point. We can add that in if there is one that we don't want to be dismissed later on. You can merge this PR if you are done with it